go.mod

```diff
@@ -105,6 +110,11 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
 
+require (
+	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358 // indirect
+	github.com/go-asn1-ber/asn1-ber v1.5.4 // indirect
+)
+
 require (
```

Comment:
If you merge these separate indirect blocks, then `go mod tidy` will sort them:
```suggestion
```

Reply:
Done 🙂